src/doctypes.js

```diff
@@ -189,11 +194,26 @@ export const triggersConn = {
   as: 'triggers'
 }
 
+export const bundleTransactionsQueryConn = ({ bundle }) => {
+  return {
+    query: () =>
+      Q(TRANSACTION_DOCTYPE)
+        .where({
+          'relationships.recurrence.data._id': bundle._id
+        })
+        .UNSAFE_noLimit()
+        .sortBy([{ date: 'desc' }])
+        .include(['bills', 'account', 'reimbursements', 'recurrence']),
+    as: `bundle-transactions-${bundle._id}`,
+    fetchPolicy: older30s
```

Comment:
Does this doctype will have a lot of data? Querying an UNSAFE_noLimit() every 30s seems a lot to me

Reply:
I do not think it is risky as it is for operations of a particular bundle. For a monthly bundle, you'd have to have 10 years of history to reach 100 operations and even then I do not think it is a performance problem.